Formula/zig.rb

```diff
@@ -2,19 +2,11 @@ class Zig < Formula
   desc "Programming language designed for robustness, optimality, and clarity"
   homepage "https://ziglang.org/"
   license "MIT"
-  revision 1
   head "https://github.com/ziglang/zig.git", branch: "master"
 
   stable do
-    url "https://ziglang.org/download/0.8.0/zig-0.8.0.tar.xz"
-    sha256 "03a828d00c06b2e3bb8b7ff706997fd76bf32503b08d759756155b6e8c981e77"
-
-    # Fix compilation of C code on Mojave. Remove at version bump.
-    # https://github.com/ziglang/zig/pull/9427
-    patch do
-      url "https://github.com/ziglang/zig/commit/24bfd7bdddbf045c5568c1bb67a3f754c24eb8c4.patch?full_index=1"
-      sha256 "feda7d03502c073bd9874996453da6961dcf16f5a3e08b86d6df1d4cbc1475a7"
-    end
+    url "https://ziglang.org/download/0.8.1/zig-0.8.1.tar.xz"
+    sha256 "8c428e14a0a89cb7a15a6768424a37442292858cdb695e2eb503fa3c7bf47f1a"
   end
```

Comment:
Let's remove the `stable` block too since it's no longer needed. The `url` and `sha256` can be declared at the top.